lib/stylesheet/importer.rb

```diff
@@ -61,7 +61,29 @@ def self.register_imports!
       register_import "wizard_fonts" do
         contents = +""
 
+        # this ensures system font can be used in wizard canvas preview
+        # technique via: https://github.com/jonathantneal/system-font-css
+        contents << <<~EOF
+          @font-face {
+            font-family: system;
+            font-style: normal;
+            src: local(".SFNS-Regular"), local(".SFNSText-Regular"), local(".HelveticaNeueDeskInterface-Regular"), local(".LucidaGrandeUI"), local("Segoe UI"), local("Ubuntu"), local("Roboto-Regular"), local("DroidSans"), local("Tahoma");
+            font-weight: 400;
+          }
+          @font-face {
+            font-family: system;
+            font-style: normal;
+            font-weight: 700;
+            src: local(".SFNS-Bold"), local(".SFNSText-Bold"), local(".HelveticaNeueDeskInterface-Bold"), local(".LucidaGrandeUI"), local("Segoe UI Bold"), local("Ubuntu Bold"), local("Roboto-Bold"), local("DroidSans-Bold"), local("Tahoma Bold");
+          }
+          .font-system {
+            font-family: system;
+          }
+        EOF
+
         DiscourseFonts.fonts.each do |font|
+          next if font[:key] == "system"
```

Comment:
Can you get rid of the code duplication above with a special handling for the "system" font here and some changes to `font_css` below (to allow arbitrary `src` values)?

```suggestion
          if font[:key] == "system"
            # Overwrite font definition because the preview canvases in the wizard do not like "-apple-system" and require explicit @font-face.
            font = {
              name: "System",
              key: "system",
              stack: "System",
              variants: [
                { src: 'local(".SFNS-Regular"), local(".SFNSText-Regular"), local(".HelveticaNeueDeskInterface-Regular"), local(".LucidaGrandeUI"), local("Segoe UI"), local("Ubuntu"), local("Roboto-Regular"), local("DroidSans"), local("Tahoma")', weight: 400 },
                { src: 'local(".SFNS-Bold"), local(".SFNSText-Bold"), local(".HelveticaNeueDeskInterface-Bold"), local(".LucidaGrandeUI"), local("Segoe UI Bold"), local("Ubuntu Bold"), local("Roboto-Bold"), local("DroidSans-Bold"), local("Tahoma Bold")', weight: 700 }
              ]
            }
          end
```